apps/webapp/src/components/dashboard/layout.tsx

```diff
@@ -13,7 +13,7 @@ export const DashboardLayout = () => {
     <div className='grid gap-6 md:gap-4 lg:grid-cols-3 xl:gap-5'>
       <Card
         gradient
-        className='order-2 flex-1 p-5 md:p-4 lg:order-1 lg:col-span-2 lg:row-span-2 xl:p-5'
+        className='order-2 flex flex-1 flex-col p-5 md:p-4 lg:order-1 lg:col-span-2 lg:row-span-2 xl:p-5'
```

Comment:
Also, taking the chance to fix that centering issue on the dashboard for the toggle